libs/ledger-live-common/src/wallet-api/constants.ts

```diff
@@ -13,7 +13,14 @@ export const WALLET_API_FAMILIES = [...FAMILIES, "evm"];
 export const WALLET_API_VERSION = "2.0.0";
 
 export const BROWSE_SEARCH_OPTIONS: Fuse.IFuseOptions<AppManifest> = {
-  keys: ["name", "categories"],
+  keys: [
+    "name",
+    "categories",
+    {
+      name: "mergedDescriptions",
+      getFn: item => Object.values(item.content.description).join(" "),
```

Comment:
I'm wondering if it might be too slow when we start having lots of translation, but I don't have a good alternative